src/main/java/io/github/lottetreg/httpserver/HTTPRequest.java

```diff
@@ -17,38 +17,42 @@
   }
 
   public String getMethod() {
-    return this.initialLine.method;
+    return this.initialLine.getMethod();
   }
 
   public String getURI() {
-    return this.initialLine.URI;
+    return this.initialLine.getURI();
   }
 
   public String getHTTPVersion() {
-    return this.initialLine.HTTPVersion;
+    return this.initialLine.getHTTPVersion();
   }
 
   public String getHeader(String header) {
     return this.headers.getHeader(header);
   }
 
   public int getContentLength() {
-    String contentLength = this.headers.getHeaderOrDefault(
+    String contentLength = this.headers.getHeader(
             "Content-Length",
             "0");
     try {
       return Integer.valueOf(contentLength);
-    } catch(NumberFormatException e) {
-      throw new InvalidContentLengthException(contentLength);
+    } catch (NumberFormatException e) {
+      throw new InvalidContentLength(contentLength); // log error but still return 0?
```

Comment:
RE: comment
If the content length header exists, but the value isn't parse-able as an int, I would just throw and return a 400BadRequest to the client, rather than just log and contunue. I'm not sure if that's what your question is though 😆

Reply:
Yes that's exactly my question! I forgot I left this comment in here... I want to create another story for 400s and come back and address this. Connor had been talking to me about Postel's Law and being liberal in what you accept but conservative in what you return. So in this case that would mean just ignoring the 'Content-Length' if it isn't formatted correctly. But I'm not sure.. I haven't made up my mind yet!